src/libs/ReportUtils.ts

```diff
@@ -2805,7 +2805,7 @@ function canHoldUnholdReportAction(reportAction: OnyxInputOrEntry<ReportAction>)
     const canModifyStatus = !isTrackExpenseMoneyReport && (isPolicyAdmin || isActionOwner || isApprover);
     const isDeletedParentAction = isEmptyObject(parentReportAction) || ReportActionsUtils.isDeletedAction(parentReportAction);
 
-    const canHoldOrUnholdRequest = !isRequestSettled && !isApproved && !isDeletedParentAction;
+    const canHoldOrUnholdRequest = !isRequestSettled && !isApproved && !isDeletedParentAction && !isClosedReport(moneyRequestReport);
```

Comment:
This was a change made because this check was added into MoneyRequestHeader and MoneyReportHeader logic for holds, and since the promoted action hold uses ReportUtils I also added it here for consistency. I would advise against reverting it here.

Reply:
@cdOut do you happen to know the background for why?

Reply:
@grgia On main there was the possiblity to hold expenses which were already closed and it was not handled, hence why the `!isClosedReport(moneyRequestReport)` was added.

Reply:
> For consistency's sake and not removing this fix from the flow of the app I would not revert this one.

https://github.com/Expensify/App/pull/44656 cc @NikkiWines 


If we keep this check, shouldn't we update all of the `canHoldOrUnholdRequest` declarations (I see 3, ReportUtils, MoneyRequestHeader, MoneyReportHeader) to include this?

Reply:
@parasharrajat

Reply:
Oh NVM it's early, probably not the `MoneyRequestHeader`. So yeah this LGTM

Reply:
I've reverted it just in case and I'll include this change in the PR fixing hold logic, should be good to merge.

cc: @grgia @parasharrajat